File: engine/model/src/main/scala/tools/packet/family/FamilyGainReputation.scala
package tools.packet.family

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class FamilyGainReputation(private var _gain: Int, private var _characterNameFrom: String) extends PacketInput {
  def gain: Int = _gain

  def characterNameFrom: String = _characterNameFrom

  override def opcode(): SendOpcode = SendOpcode.FAMILY_REP_GAIN
}

File: engine/engine-model/src/main/java/tools/packet/family/FamilyGainReputation.java
package tools.packet.family;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record FamilyGainReputation(Integer gain, String characterNameFrom) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.FAMILY_REP_GAIN;
   }
}